rock/rockcraft.yaml

```diff
@@ -147,4 +163,4 @@ parts:
         chmod +x "$CRAFT_PRIME/scripts/run-dbus.sh"
       fi
 
-    after: [ipp-usb, avahi-daemon]
+    after: [ipp-usb, avahi-daemon]
```

Comment:
Pure whitespace change. Please avoid/remove those  as they clutter the proposed code for the PR.